bin/init.js

```diff
@@ -281,12 +281,12 @@ export async function init() {
 
   // Add dependencies
   packageJson.dependencies = {
-    "@farcaster/auth-client": "^0.3.0",
-    "@farcaster/auth-kit": "^0.6.0",
-    "@farcaster/frame-core": "^0.0.29", 
-    "@farcaster/frame-node": "^0.0.18",
-    "@farcaster/frame-sdk": "^0.0.31",
-    "@farcaster/frame-wagmi-connector": "^0.0.19",
+    "@farcaster/auth-client": "latest",
+    "@farcaster/auth-kit": "latest",
+    "@farcaster/frame-core": "latest", 
+    "@farcaster/frame-node": "latest",
+    "@farcaster/frame-sdk": "latest",
+    "@farcaster/frame-wagmi-connector": "latest",
```

Comment:
`"latest"` can be dangerous, as it allows major version bumps with potentially breaking changes.

the existing `^` syntax is actually supposed to bump to the latest minor version automatically, but I guess npm can behave differently with `0.x.x` version numbers.

I'd suggest the following to pin the major version to 0 but otherwise allow the latest package:

```suggestion
    "@farcaster/auth-client": ">=0.3.0 <1.0.0",
    "@farcaster/auth-kit": ">=0.6.0 <1.0.0",
    "@farcaster/frame-core": ">=0.0.29 <1.0.0",
    "@farcaster/frame-node": ">=0.0.18 <1.0.0",
    "@farcaster/frame-sdk": ">=0.0.31 <1.0.0",
    "@farcaster/frame-wagmi-connector": ">=0.0.19 <1.0.0",
```

Reply:
Thanks for the review. Will make the changes.